templates/rhel8.saphana.tpl.yaml

```diff
@@ -15,6 +15,10 @@ metadata:
     template.openshift.io/bindable: "false"
     template.kubevirt.io/version: v1alpha1
     defaults.template.kubevirt.io/disk: rootdisk
+{% if item.workload != "desktop" %}
+    template.kubevirt.io/containerdisks: |
+      registry.redhat.io/rhel8/rhel-guest-image
```

Comment:
@dominikholler do we want to use this link for sap hana template?

Reply:
no, hana template has the SRC_CONTAINERDISK parameter already.